js/components/EditProfileForm/EditProfileForm.js

```diff
@@ -4,88 +4,132 @@ import {
   Text,
   TextInput,
   TouchableOpacity,
-  StyleSheet
+  StyleSheet,
 } from "react-native";
 import getAPI from "../../config/api";
 import { Form, Field } from "react-final-form";
 import LinearGradient from "react-native-linear-gradient";
+import moment from "moment";
 import globalStyles from "../../config/styles";
 import styles from "./styles";
 
 export default class EditProfileForm extends Component {
   constructor(props) {
     super(props);
+    this.state = {
+      initialValues: {}
+    };
   }
+  componentDidMount() {
+    this.getUser();
+  }
+
+  getUser = async values => {
+    const token =
+      "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwczpcL1wvYXBpLnNwZWFrYm94LXN0YWdpbmcuY2FcL2FwaVwvYXV0aFwvcmVnaXN0ZXIiLCJpYXQiOjE1NDQ3NDQ3OTQsImV4cCI6MTU0NDc0ODM5NCwibmJmIjoxNTQ0NzQ0Nzk0LCJqdGkiOiJ3aFJiUW1iMDNwSHpOZFJRIiwic3ViIjo3LCJwcnYiOiI4N2UwYWYxZWY5ZmQxNTgxMmZkZWM5NzE1M2ExNGUwYjA0NzU0NmFhIn0.04EaH262WbiN-Ss1p1ABDaYo3dfLu1431YGd-3zg7_s";
```

Comment:
if you are doing `this.getUser()` you should not be hardcoding your token.

Reply:
I removed the tokens and will implement this.getUser() once this pull request is approved.